frontend/src/containers/AdmissionsContainer/index.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useMemo } from "react";
+import { useTable, useSortBy, useExpanded } from "react-table";
+import { cols } from "./Columns";
+import AdmissionsInnerTable from "./InnerTable";
+import SubComponentWrapper from "./SubComponentWrapper";
+import SubComponentHeader from "./SubComponentHeader";
+
+const AdmissionsContainer = ({ applications }) => {
+  const columns = useMemo(() => cols, []);
+  const data = useMemo(
+    () =>
+      applications.map((application) => ({
+        username: application.user.username,
+        fullname: application.user.full_name,
+        phone_number: application.phone_number,
+        email: application.user.email,
+        applied_within_deadline: application.applied_within_deadline,
+        numApplications: application.group_applications.length,
+        created_at: application.created_at,
+        updated_at: application.updated_at,
+        text: application.text,
+        groupApplications: application.group_applications.map(
+          (application) => ({
+            group: application.group.name,
+            text: application.text,
+          })
+        ),
+      })),
```

Comment:
As you change some of the variable names into camelCase, it'd be awesome if you could change all of them:)

Reply:
yes i agree, i just used whatever was in the applications object and some of the variable names was camelcase and some were snakecase, but i think i'll stick to camelcase where i can :)